src/Nexus.UI/Controls/UIDatePicker.razor

```diff
@@ -20,20 +20,17 @@
 
     private DateTime _tmp;
 
-    private DateTime _value;
-
     [Parameter]
-    public DateTime Value
+    public DateTime Value {get; set;}
+
+    protected override void OnParametersSet()
```

Comment:
Methods should be sorted after properties (and after constructors, but there is no constructor)